presto-hudi/pom.xml

```diff
@@ -44,11 +44,6 @@
             <artifactId>javax.inject</artifactId>
         </dependency>
 
-        <dependency>
```

Comment:
Do we have any plan to remove joda-time totally from presto or presto-hive? 

joda-time is used in presto-hive which is depended by presto-hudi.

Reply:
Eventually, we want to get rid of joda since the joda library is now explicitly asking users to migrate to java.time.

from https://www.joda.org/joda-time/
> Note that from Java SE 8 onwards, users are asked to migrate to java.time (JSR-310) - a core part of the JDK which replaces this project.